### ISSUE TITLE
Automatically Select Language based on URL parameter 

### ISSUE THREAD

**Issue by Davinkjohnson** [episphere/connect#1086]
Add a URL parameter that pre-populates the language selection e.g., myconnect.cancer.gov/?lang=es that then gets parsed to change the value of the language selection.

**Comment by sonyekere** [comment]
Hi @robertsamm #1105 address some of that Spanish work. If this issues is still relevant, do we have a milestone in mind?

**Comment by depietrodeanna** [comment]
Yes let's work on this. Some time during summer would be OK. I am assigning this July release but that is flexible. @sonyekere

**Comment by bransteitterbr** [comment]
This is merged into dev and ready for testing

**Comment by depietrodeanna** [comment]
@bransteitterbr could you let me know if this requires URL updates on my end before testing? Or how I should go about testing (do I just get a Dev email for someone who has Spanish as preferred language and click the link for MyConnect and it should pop up as the Spanish version of the PWA?

**Comment by bransteitterbr** [comment]
@depietrodeanna I have updated the login link generated by the email login flow to include the new `lang` parameter.  As for other links in the communications they will have to be updated so that the Spanish version includes `lang=es` as part of the url

**Comment by depietrodeanna** [comment]
OK, tested the login email link flow and that works in Dev. @bransteitterbr

**Comment by depietrodeanna** [comment]
@bransteitterbr I am going to test an update to one of the URLs in a Spanish email. I'm modifying the first notification that someone receives right after they are verified. Could you help me format the correct URL for this example Dev link to MyConnect? Where does lang=es go?

<a href="https://nci-c4cp.github.io/connectApp/" target="_blank">

**Comment by depietrodeanna** [comment]
this has been tested in Dev and ready for Stage

**Comment by sonyekere** [comment]
@depietrodeanna Is this ready for prod?

**Comment by depietrodeanna** [comment]
Yes @sonyekere sorry I was far too in the weeds on diet screener testing and other high priority items this week and neglected to update. For completeness, adding here that the URL update this GH issue entails automatically populates the verification email for login based on the language shown on the PWA landing page. This does **not** change the language of the login email based on preferred language. All is functioning in Stage.

**Comment by jwiernusz** [comment]
As built
